lms/templates/learner_dashboard/course_card.underscore

```diff
@@ -3,14 +3,17 @@
         <div class="course-image-container">
             <% if (course_url){ %>
                 <a href="<%- course_url %>" class="course-image-link">
-            <% } %>
+                    <img
+                        class="header-img"
+                        src="<%- course_image_url %>"
+                        alt="<%= interpolate(gettext('%(courseName)s Home Page.'), {courseName: display_name}, true) %>"/>
+                </a>
+            <% } else { %>
                 <img
                     class="header-img"
-                    src="<%- course_image_url %>"
-                    alt="<%= interpolate(gettext('%(courseName)s Home Page.'), {courseName: display_name}, true) %>"/>
-            <% if (course_url){ %>
-                </a>
+                    src="<%- course_image_url %>" />
```

Comment:
I think that this image is purely decorative if it is not a link, I think it would make sense to give it a blank alt (so it is hidden from screenreaders). As it currently stands it is invalid HTML because all images should have an alt attribute.